api/pipeline.go

```diff
@@ -0,0 +1,647 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package api
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/go-vela/compiler/compiler"
+
+	"github.com/go-vela/server/database"
+	"github.com/go-vela/server/router/middleware/repo"
+	"github.com/go-vela/server/source"
+	"github.com/go-vela/server/util"
+
+	"github.com/go-vela/types"
+	"github.com/go-vela/types/yaml"
+
+	"github.com/gin-gonic/gin"
+)
+
+const (
+	outputJSON = "json"
+	outputYAML = "yaml"
+)
+
+// swagger:operation GET /api/v1/pipelines/{org}/{repo} pipelines GetPipeline
+//
+// Get a pipeline configuration from the source provider
+//
+// ---
+// x-success_http_code: '200'
+// produces:
+// - application/json
```

Comment:
By default, this endpoint produces YAML rather than JSON.

I think we'll want to make sure this is properly reflected in the API spec 👍 

```suggestion
// produces:
// - application/x-yaml
// - application/json
```

Reply:
done 👍